### DISCUSSION TITLE
Include `name` property in outputs of `page.images`

### DISCUSSION THREAD

**Discussion by djr2015** [jsvine/pdfplumber#1201] [Show and tell]
When working with images inside pdfs, how about also recording the object's id when computing `page.images` via:
`attr["name"]=obj.name`
here: https://github.com/jsvine/pdfplumber/blob/stable/pdfplumber/page.py#L369

Doing this enables mapping images' coordinates and dimensions to their identifiers as recorded in the pdf (e.g. `/Im0`) which is useful for downstream tasks involving other pdf parsing libraries.

**Comment by jsvine** [comment]
Thanks! That's a relatively straightforward add, now available on `develop` and will be merged into the next release: https://github.com/jsvine/pdfplumber/commit/6ef62c97914a9c88a889875cb1af710c1da439d1
